### PR TITLE
Loki Variable Query Editor: Improve stream selector tooltip text

### PR DESCRIPTION
**Pull request by matyax**
**Why do we need this feature?**

When creating variables that query for label values the users can input a stream selector. A user had troubles trying to use a stream selector with interpolation, thinking it was because it was not supported. The actual problem was caused for the actual stream selector, which needs to be in the correct format. For example: `{name="value"}` or `{name="$interpolated"}`.

This PR improves the tooltip to remind users of the expected format of a stream selector.

**Which issue(s) does this PR fix?**:

Related with https://github.com/grafana/grafana/issues/52380

**Special notes for your reviewer**:

See the tooltip, understand it.

imagen

### CODE REVIEW

File: public/app/plugins/datasource/loki/components/VariableQueryEditor.tsx [hunk 1]
```diff
@@ -103,7 +103,9 @@ export const LokiVariableQueryEditor: FC<Props> = ({ onChange, query, datasource
             labelWidth={20}
             tooltip={
               <div>
-                Optional. If defined, a list of values for the label in the specified log stream selector is returned.
+                {
+                  'Optional. If defined, a list of values for the label in the specified log stream selector is returned. E.g. {stream="name"}'
```
Comment (svennergr) [inline public/app/plugins/datasource/loki/components/VariableQueryEditor.tsx:107]: Just a very nit, but I think in Loki denomination a `stream` can also be a combination of labels. WDYT changing `stream` to `label` and `name` to `value`? That would also be in line with the empty stream selectors in Query Builder: 
image

Comment (matyax) [reply public/app/plugins/datasource/loki/components/VariableQueryEditor.tsx:107]: I don't really have an opinion about that, so I'd go with whatever you think is better.

Looping @ivanahuckova

Comment (matyax) [reply public/app/plugins/datasource/loki/components/VariableQueryEditor.tsx:107]: Updated.

**Review by ivanahuckova — APPROVED** [review]

File: public/app/plugins/datasource/loki/components/VariableQueryEditor.tsx (hunk 1 above)
Comment (svennergr) [reply public/app/plugins/datasource/loki/components/VariableQueryEditor.tsx]: I would've liked `{label=value}` the most, but `{name=value}` is also fine I guess 👍

**Review by svennergr — APPROVED** [review]

File: public/app/plugins/datasource/loki/components/VariableQueryEditor.tsx (hunk 1 above)
Comment (matyax) [reply public/app/plugins/datasource/loki/components/VariableQueryEditor.tsx]: I  actually misread your comment. Updated again.

**Review by svennergr — APPROVED** [review]
Alt Text

**Comment by ivanahuckova** [comment]
One more idea, I like how you in PR description also mention interpolated labels. Maybe we could also add that to example. To make it even more clear we interpolate variables in variables editor.

**Comment by matyax** [comment]
I like the idea. Updated again with a minor simplification to the text.

**Pull request merged**